app/assets/javascripts/charts.js

```diff
@@ -259,7 +260,12 @@ function get_pie_chart(div, url) {
       $.getJSON(url, function(data) {
         var target = $("#"+div+"-body");
         target.empty();
+        var hostsCount = 0;
+        $.each(data.values,function() {
+          hostsCount += this.data;
+        });
         expanded_pie(target, data.values);
+        $('.modal-title').empty().append( __('Fact distribution chart') + ' - <b>' + data.name + ' </b><small> ('+ Jed.sprintf(n__("%s host", "%s hosts", hostsCount), hostsCount) +')</small>');
```

Comment:
somehow, we are still missing the number inside the pie chart? any idea where did it go?